x-pack/test/apm_api_integration/tests/alerts/transaction_error_rate.spec.ts

```diff
@@ -82,38 +66,19 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
     after(async () => {
       await synthtraceEsClient.clean();
-      await supertest.delete(`/api/alerting/rule/${ruleId1}`).set('kbn-xsrf', 'foo');
-      await supertest.delete(`/api/actions/connector/${actionId1}`).set('kbn-xsrf', 'foo');
-      await supertest.delete(`/api/alerting/rule/${ruleId2}`).set('kbn-xsrf', 'foo');
-      await supertest.delete(`/api/actions/connector/${actionId2}`).set('kbn-xsrf', 'foo');
```

Comment:
This is problematic: rules and alerts are created per test but not deleted until all tests have run. This means that the rules and alerts created in the first test still exists when running the second test. This means that tests were required to run in a specific order, and it was not possible to run tests in isolation.